docs/tails_guide.rst

```diff
@@ -1,72 +1,70 @@
+access the *Document Interface* and create new users.
+
+If you followed the :doc:`SecureDrop Installation instructions <install>` 
+correctly, you should have already created a *journalist Tails USB* and an 
+*admin Tails USB* and enabled the persistence volume on each. If you have not, 
+or need to create another *journalist Tails USB* for a second journalist, 
+follow the steps below.
+
+If you already know how to boot the *admin Tails USB* or the *journalist Tails 
+USB* with persistence, you can skip down to the step 'download the repository'.
+
+Note that for all of these instructions to work, you should have already 
 installed the main SecureDrop application.
 
 Installing Tails on USB sticks
 ------------------------------
 
-Tails is a live operating system that is run from removable media, such
-as a DVD or a USB stick. For SecureDrop, you'll need to install Tails
-onto USB sticks and enable persistent storage.
+Tails is a live operating system that is run from removable media, such as a 
+DVD or a USB stick. For SecureDrop, you'll need to install Tails onto USB 
+sticks and enable persistent storage.
 
-We recommend creating an initial Tails Live USB or DVD, and then using
-that to create additional Tails Live USBs with the *Tails Installer*, a
-special program that is only available from inside Tails. *You will only
-be able to create persistent volumes on USB sticks that had Tails
-installed via the Tails Installer*.
+We recommend creating an initial Tails Live USB or DVD, and then using that to 
+create additional Tails Live USBs with the *Tails Installer*, a special program 
+that is only available from inside Tails. *You will only be able to create 
+persistent volumes on USB sticks that had Tails installed via the Tails 
+Installer*.
 
-The `Tails website <https://tails.boum.org/>`__ has detailed and
-up-to-date instructions on how to download and verify Tails, and how to
-create a Tails USB stick. Here are some links to help you out:
+The `Tails website <https://tails.boum.org/>`__ has detailed and up-to-date 
+instructions on how to download and verify Tails, and how to create a Tails USB 
+stick. Here are some links to help you out:
 
--  `Download and verify the Tails
-   .iso <https://tails.boum.org/download/index.en.html>`__
--  `Install onto a USB stick or SD
-   card <https://tails.boum.org/doc/first_steps/installation/index.en.html>`__
--  `Create & configure the persistent
-   volume <https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html>`__
+.. _`Download and verify the Tails .iso`: https://tails.boum.org/download/index.en.html
+.. _`Install onto a USB stick or SD card`: https://tails.boum.org/doc/first_steps/installation/index.en.html
+.. _`Create & configure the persistent volume`: https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html
+.. _`manually installing Tails onto a USB device on Mac OS X`: https://tails.boum.org/doc/first_steps/installation/manual/mac/index.en.html
```

Comment:
The references should be defined _after_ they are used, not before. Look at any of the other docs to see how this is done.